tools/project_manager/show_good_word_suggestions_detail.php

```diff
@@ -45,109 +46,100 @@
-    echo "<h2>", 
-        // TRANSLATORS: %1$s is a word and %2$s is a project name.
-        sprintf(_("Suggestion context for '%1\$s' in %2\$s"),
-            $word, $project_name),
-        "</h2>";
-
-    echo "<p>";
-    echo "<a href='show_word_context.php?projectid=$projectid&amp;word=$encWord' target='_PARENT'>" .
-         _("Show full context set for this word") . "</a>";
-
-    echo " | ";
-    echo "<a target='_PARENT' href='" . attr_safe($_SERVER['PHP_SELF']) . "?projectid=$projectid&amp;word=$encWord&amp;timeCutoff=$timeCutoff&amp;";
-    if($layout == LAYOUT_HORIZ)
-        echo "layout=" . LAYOUT_VERT . "'>" . _("Change to vertical layout");
-    else
-        echo "layout=" . LAYOUT_HORIZ . "'>" . _("Change to horizontal layout");
-    echo "</a>";
-    echo "</p>";
-
-    foreach($word_suggestions as $suggestion) {
-        list($time,$round,$page,$proofer,$words)=$suggestion;
-        // get a context string
-        list($context_strings,$totalLines)=_get_word_context_on_page($projectid,$page,$round,$word);
-
-        # If the word was suggested on a page, but then changed before
-        # being saved, let the PM know about it.
-        if(!count($context_strings))
-        {
-            echo "<p>" . sprintf(_('The word was suggested in round %1$s for page %2$s, but no longer exists in the saved text for that round.'), $round, $page) . "</p>";
-            continue;
-        }
-
-        echo "<p><b>" . _("Date") . "</b>: " . strftime($datetime_format,$time) . "<br>";
-        echo "<b>" . _("Round") . "</b>: $round &nbsp; | &nbsp; ";
-        echo "<b>" . _("Proofreader") . "</b>: " . private_message_link($proofer) . "<br>";
-        echo "<b>" . _("Page") . "</b>: <a href='../page_browser.php?simpleHeader=true&amp;project=$projectid&amp;imagefile=$page' target='imageframe'>$page</a><br>";
-        foreach($context_strings as $lineNum => $context_string) {
-            $context_string=_highlight_word(html_safe($context_string, ENT_NOQUOTES), $word);
-            echo "<b>" . _("Line") . "</b>: ", 
-                // TRANSLATORS: %1$d is the approximate line number, and 
-                // %2$d is the total number of lines when displaying the 
-                // context of a word.
-                sprintf(_('~%1$d of %2$d'), $lineNum, $totalLines),
-                " &nbsp; | &nbsp; ";
-            echo "<b>" . _("Context") . "</b>:<br><span class='mono'>$context_string</span><br>";
-        }
-        echo "</p>";
-        echo "<hr>";
-
+    echo "<p><b>" . _("Date") . "</b>: " . strftime($datetime_format,$time) . "<br>";
+    echo "<b>" . _("Round") . "</b>: $round &nbsp; | &nbsp; ";
+    echo "<b>" . _("Proofreader") . "</b>: " . private_message_link($proofer) . "<br>";
+    echo "<b>" . _("Page") . "</b>: <a href='$code_url/tools/project_manager/show_good_word_suggestions_detail.php?projectid=$projectid&amp;imagefile=$page&amp;word=$encWord&amp;timeCutoff=$timeCutoff'>$page</a><br>";
```

Comment:
Since this is self-referencing I think `<a href='?projectid=$projectid...>` should work here too.

Reply:
I'll fix these and the eslint errors in build tonight.